pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -664,6 +664,8 @@ const (
 
 	// SeedNginxIngressClass defines the ingress class for the seed nginx ingress controller
 	SeedNginxIngressClass = "nginx-ingress-gardener"
+	// NginxIngressClass defines the ingress class for the shoot nginx ingress controller addon.
+	NginxIngressClass = "nginx"
```

Comment:
```suggestion
	ShootNginxIngressClass = "nginx"
```

to make it more clear?